Makefile

```diff
@@ -28,6 +28,11 @@ clean:
 	@echo "--> Cleaning up ./build"
 	@rm -rf build/*
 
+## clean-cache: removes all go cache, useful for when your tests aren't passing due to cached old code
```

Comment:
Not sure I get the use case. How tests cannot pass because of old cached code? I've never encountered this. Maybe there is another solution to the problem?

Reply:
Prior to go1.18, I've used to run into issues such as not seeing a debug line I've put in a test or in a module being tested, and pruning the cache made those lines appear, and while I was making my way with this PR, I had to run this line to remove any doubts.

I agree, the use-case is not well-defined. (I don't feel strongly about this addition)

Reply:
Actually, I've just ran into this with the `das` package. Maybe we can discuss this and ship it in another PR.